src/user/bans.js

```diff
@@ -126,13 +126,13 @@ module.exports = function (User) {
 		], callback);
 	};
 
+	// TODO Remove in v1.13.0
 	const deprecate = function (func, oldPath, newPath) {
```

Comment:
There's a built-in deprecate function in node util module, maybe worth considering

Reply:
TIL, thanks for mentioning it @pitaj